build.sbt

```diff
@@ -279,5 +279,3 @@ lazy val root = (project in file("."))
   .aggregate(wdlTransformsDraft3)
   .aggregate(wom)
   .aggregate(womtool)
-  // TODO: See comment in plugins.sbt regarding SBT 1.x
-  .enablePlugins(CrossPerProjectPlugin)
```

Comment:
Is this a real change or just no longer required?

Reply:
It's part of the sbt "doge" plugin (helps for building against multiple scala versions) that was merged into SBT proper as of 1.x.  Hence the plugin is no longer needed.